mne/viz/_3d.py

```diff
@@ -470,6 +490,126 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     -------
     fig : instance of mlab.Figure
         The mayavi figure.
+
+    See Also
+    --------
+    mne.viz.plot_bem
+    """
+    return plot_alignment(info, trans=trans, subject=subject,
+                          subjects_dir=subjects_dir, source=source,
+                          coord_frame=coord_frame, meg_sensors=meg_sensors,
+                          eeg_sensors=eeg_sensors, dig=dig, ref_meg=ref_meg,
+                          ecog_sensors=ecog_sensors, head=head, brain=brain,
+                          skull=skull, src=src, mri_fiducials=mri_fiducials,
+                          bem=bem, verbose=verbose)
+
+
+@verbose
+def plot_alignment(info, trans='auto', subject=None, subjects_dir=None,
```

Comment:
Looking at it, maybe we should keep ``brain``. There are so many options for different kinds of brain surfaces that it would be a pain to parse the surfaces. wdyt?